GUIDE.md

```diff
@@ -63,7 +57,7 @@ As `$USER` user with `sudo` privileges,
 
 1. A popup window labeled `New Bare Metal OpenShift Cluster` opens and requests a Cluster Name and OpenShift Version. Enter an appropriate Cluster Name.
 
-   > NOTE: The OpenShift Version selected is the value assigned to `OPENSHIFT_VERSION` (defaults to 4.7)
+   > NOTE: The OpenShift Version selected is the value assigned to `OPENSHIFT_VERSION` (defaults to 4.9)
```

Comment:
Maybe it's better to specify that the default is taken from assisted service `openshift_versions`